src/lib/components/schedule/schedules-count.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import Skeleton from '$lib/holocene/skeleton/index.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { fetchScheduleCount } from '$lib/services/workflow-counts';
+  import { schedulesCount } from '$lib/stores/schedules';
+
+  $: namespace = $page.params.namespace;
+
+  let loading = false;
+
+  const fetchCounts = async () => {
+    loading = true;
+    try {
+      $schedulesCount = await fetchScheduleCount({
+        namespace,
+      });
+    } catch (e) {
+      console.error('Fetching schedules count failed: ', e?.message);
+    } finally {
+      loading = false;
+    }
+  };
+
+  $: namespace, fetchCounts();
+</script>
+
+<div class="flex flex-wrap items-center gap-2">
+  {#if !loading}
+    {$schedulesCount}
+  {:else}
+    <Skeleton class="h-6 w-6 rounded" />
+  {/if}
+  {translate('common.schedules')}
```

Comment:
What do you think about doing something similar to what we have for `Workflows` here and adding a plural translation? E.g. updating this ⬇️ 

```suggestion
  <Translate key="common.schedules-plural" count={Number($schedulesCount)} />
```
and adding the following to `common.ts`
```js
  'schedules-plural_one': 'Schedule',
  'schedules-plural_other': 'Schedules',
```

so that we have `1 Schedule` instead of `1 Schedules`

<img width="599" alt="Screenshot 2024-06-11 at 3 01 50 PM" src="https://github.com/temporalio/ui/assets/15069288/1942551a-28ff-46af-a155-72fc95df6a72">

Reply:
👍